README.md

```diff
@@ -88,25 +88,34 @@ To help you get started with the functionalities provided by this library, the
 tutorial covers how to train a MNIST model using TensorFlow,
 craft adversarial examples using the [fast gradient sign method](https://arxiv.org/abs/1412.6572),
 and make the model more robust to adversarial examples using adversarial training.
+* **MNIST with CW** ([code](cleverhans_tutorials/mnist_tutorial_cw.py)): this
+tutorial covers how to train a MNIST model using TensorFlow, and
+craft adversarial examples using the [C&W attack](https://arxiv.org/abs/1608.04644).
+* **MNIST with EAD** ([code](cleverhans_tutorials/mnist_tutorial_ead.py)): this
```

Comment:
As commented below, we probably won't want this to be in the `tutorials` directory, so after resolving that issue we'll need to update this text.